students/christopher_gantt/lesson03/mailroom.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+donor_db = [("Wassily Kandinsky", [43928.13, 131.34, 1928.0]),
+            ("Jasper Johns", [3134.43, 153.34]),
+            ("Mark Rothko", [135353.33]),
+            ("Richard Serra", [153757.87, 28457.12, 1293451.0]),
+            ("Yves Tanguy", [1534.23, 2542.19])]
+
+prompt = "\n".join(("Type a number from the options below",
+        "1 - Send a Thank You",
+        "2 - Create a Report",
+        "3 - Quit\n"))
+
+def thank_you():
+    thanks = True
+    while thanks == True:
```

Comment:
use `is` instead of `==`

`is` is used for object comparison (Ex. `is None` `is True`)

however, here you can just simplify to `while thanks:`

Reply:
Thanks for the feedback Natasha